code/datums/martial/wrestling.dm

```diff
@@ -309,9 +316,7 @@
 			A.setDir(turn(A.dir, 90))
 
 		A.forceMove(D.loc)
-		spawn (4)
-			if (A && (T && isturf(T) && get_dist(A, T) <= 1))
-				A.forceMove(T)
+		addtimer(CALLBACK(src, .proc/CheckStrikeTurf, A, T), 4)
```

Comment:
no.

```.``` has a special meaning when attached to types, it's something like "search up or down the type hierarchy to find this"